examples/OCPS.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
<p>Is there information somewhere on understanding the output? Linking that would be useful. For example, we couldn't figure out where the output was written and if something went into the butler how we'd get it back.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
I do need to write more documentation.  But outputs are tricky.  The original intent was to have them be a few scalar numbers (e.g. PSF FWHM and ellipticities or photometric zeropoint or computed boresight location).  These would be retrieved from the back-end UWS service by the OCPS and then returned as straightforward key/value JSON in the result message.  But I have no examples of such scalars from the pipelines folks as of yet, so it's hard to make sure this will work properly.

Non-scalar outputs like calibration images were supposed to only be side-effects.  I had supposed that the OCPS commander would have sufficient DataId information to look them up in the Butler independently (since such information had to be supplied to the OCPS in the first place).

The current output of raw pathnames from the back-end service is mostly a placeholder that is of little utility to either use case.